### ISSUE TITLE
Weather Station not populating Sensors

### ISSUE THREAD

**Issue by guarddog13** [neilenns/ambientweather2mqtt#92]
I have a WS-2902C. I've followed the directions to set this up on my awnet app and its not populating the sensors. I'm able to see on MQTT explorer that it comes online but no sensors are populating. I have it working with a HASC integration called AWNET 2 local that works well but it requires an addon and I'm trying to move away from supervised to Core so wanted to utilize this integration with MQTT.

**Comment by neilenns** [comment]
Add `VERBOSE=true` to your `.env` file and restart the Docker container. Then check the logs and see if it shows any incoming traffic from your weather station.

If no it means the awnet config is either incorrect or your firewall rules are blocking communication between the weather station and your Docker container.

If yes then communication is working and it is something else. In that case post the log messages here so I can peek at them.

**Comment by guarddog13** [comment]
I ended up getting it setup using a python script a gentleman on HA community provided to me. Its running as a service in a very low overhead ubuntu LXC...i made the sensors in HA and its doing what I expect so i think I'm good. I might try it again later so I'll bookmark this. 

Now to figure out how to create a feels like sensor, the addon version has one and I used it lol...thinking polishing up on my template skills might be on order lol.
